subsys/bluetooth/audio/pacs.c

```diff
@@ -550,36 +608,54 @@ BT_GATT_SERVICE_DEFINE(pacs_svc,
 		      src_read, NULL, NULL),
 	BT_AUDIO_CCC(src_cfg_changed),
 #if defined(CONFIG_BT_PAC_SRC_LOC)
-#if defined(CONFIG_BT_PAC_SRC_LOC_WRITEABLE)
+#if defined(CONFIG_BT_PAC_SRC_LOC_WRITEABLE) && defined(CONFIG_BT_PAC_SRC_LOC_NOTIFIABLE)
```

Comment:
Can't comment on unmodified lines, but `BT_UUID_PACS_SRC` and `BT_UUID_PACS_SNK` are also optional notifiable.